Implement --votes=0 setting to vote all options  even if adding an option (#506)
matterpoll/matterpoll: README.md

- `--anonymous`: Don't show who voted for what at the end
- `--progress`: During the poll, show how many votes each answer option got and, in post card, show who voted for which answers ([#431](https://github.com/matterpoll/matterpoll/pull/431))
- `--public-add-option`: Allow all users to add additional options

> - `--votes=X`: Allow users to vote for X options. Default is 1. If X is 0, users can vote for all options even after adding options.

Reviewer: I wonder if the wording would be more clear if it's said that the number of votes is unlimited. 
```suggestion
- `--votes=X`: Allow users to vote for X options. Default is 1. If X is 0, users have an unlimited amount of votes.
```
